### DISCUSSION TITLE
1

### DISCUSSION THREAD

**Discussion by samaramco** [WeAreMahsaAmini/Core#353] [Ideas]
1

**Comment by AnonymousMaximus2** [comment]
این خوبه ولی پولهای اصلی دست اینها نیست، آقا زاده ها همه خورده پاها هستند


هیچ وقت اختلاسهای بزرگ را نمیگذارند مردم اصلا فکرش را بکنند.


حتی افرادی مانند بابک زنجانی و محمود خاوری توی کانادا و شهرام جزایری هم با اختلاسهای میلیارد دلاری شان درمقابل حقایق پشت پرده اعداد خورد هستند.

حساب کنید در طول 40 سال روزانه 5 میلیون بشکه نفت تولید شده جمعا میشود 73 میلیارد بشکه نفت، میانگین نفت یک زمانی دوره احمدی نژاد به 150 دلار رسیده. و زمانی به زیر 100 دلار رسیده و الان 110 دلار است میانگین فرض کنیم 100 دلار، میشود 7.3  تریلیون دلار، 

اقا زاده ها هر کدام چند میلیون دلار داشته باشند هزار نفر آقا زاده جمعشان میشود چند میلیارد دلار، معادل اختلاس خاوری، 
و کل اختلاس بابک زنجانی هم چند میلیارد دلار بود. شهرام جزایری هم که عددی نبود جوجه بود،  

دکل نفتی که گم شد قیمتش در حدود 100 میلیون دلار بود.

در طول 40 سال به غیر از نفت، دولت ایران شرکتهای دولتی، بانکها، پتروشیمی ها و معادن طلا و معادن آهن و .... همه را استخراج کرده و فروخته، فقط سالیانه 50 میلیارد دلار پتروشیمی تولید میکنه، که توی 40 سال میشه 2 تریلیون دلار، و اگر فرض کنیم 700 میلیارد دلار صادرات آهن و معادن و طلا و جواهرات قیمتی و سنگهای معدنی مانند مرمر و درآمدهای ارزی بانک مرکزی باشد، جمعا در طول 40 سال 10 تریلیون دلار ثروت مردم ایران را به جیب زده

به غیر از این 10 تریلیون دلار ذخایر نفت و گاز و معادن که زیر زمین هست حدود 150 تریلیون دلار است.

این ثروت را رها کرده اید به خورده پاها میچسبید؟

البته آقا زاده ها و فرزندان مسئولین را باید پیگیری کرد

ولی عده ای ناشناس توی دنیا هستند که این 10 تریلیون دلار ثروت مردم ایران را دارند دست به دست میکنند 


هر چند وقت یک بار یه جائی بلوکه میشه و بعد از چند سال آزاد میشه

توی اخبار چند سال پیش زمان امضای قرارداد برجام اعلام شد توی ترکیه آمریکا 100 میلیارد دلار بلوکه شده را آزاد کرده.

یا چند سال پیش 9 میلیارد دلار توی هند بلوکه شده بود.

یا توی کره جنوبی چندین میلیارد دلار بلوکه شده 

یک نکته مهم اینکه این پولها توی بانکهای بزرگ دنیا در گردش هستند،

هر پولی در هر بانکی باشد میتوان تا 10 برابر کردیت و اعتبار و وام گرفت.

یعنی این مافیای بین المللی دولت ایران اعتبار و کردیت و وامی حدود 100 تریلیون دلار دارد 

عدد یک تریلیون دلار  آنقدر بزرگ است که توی عقل و مغز ما جا نمیشود.

تبلیغات بین المللی کرده اند که خانواده یهودی روچیلد ثروتشان 2 تریلیون دلار است.

ولی کافر های جمهوری اسلامی که هیچ بوئی از اسلام و انسانیت نبرده اند، توی دست شان 10 تریلیون دلار پول نقد و 100 تریلیون دلار کردیت بانکی دارند، و مردم داخل ایران از گرسنگی میمیرند. 

هر چند گاهی از سود این ثروت عظیم مبلغ کم چند صد میلیون دلار به حزب الله لبنان یا یمن یا سوریه یا سایر کشورها کمک میکنند 

به چین و روسیه باج میدهند و حتی پول خرید هواپیماهای بدون سرنشین روسیه را میدهند و در جنگهای دنیا اسپانسور میشوند. 

دنبال اینها باشید، پشت اینها آقازاده ها و فرندان مسئولین نیستند، 


بگردید ببینید پشت اینها چه کسانی هستند

و البته هر کسی پشت این چیزها باشه نمیگذارند زنده بماند و هر جای دنیا باشد ترورش میکنند

این پولها برای بازی قمار و شطرنج با کشورهای روسیه و چین در مقابل آمریکا و اروپا است.

و با کسی هم شوخی ندارند 

من بیشتر از اینها نمیتوانم حرف بزنم سر نخ را دادم خودتان دنبالش بروید اگر جراتش را دارید

من جرات بیشتر از این حرف زدن را ندارم.

**Reply by samaramco** [reply]
بزرگوار ممنون از توضیحات 
چیزهایی که شما می‌گوید را من بارها در ایران از زبان راننده تاکسی و .... شنیده ام 
سخن من کارزاریست که فرزندان مسئولان که در کشورهای جهان هستند پاسخگو کنیم به سوالات 

اینگونه وقتی مسئول می‌بیند فرزندش زیر فشار رسانه ای قرار دارد مجبور که کاهش فشار و این کاهش فشار داخلی موجب می‌شود برادر و خواهر ما در ایران بهتر بتواند به ادامه اعتراض ها بپردازد ...
ما در خارج از کشور 
راه هایی که داریم را امتحان کنیم برای کاهش فشاربر مردم داخل کشور

**Reply by AnonymousMaximus2** [reply]
شما از زبان راننده تاکسی شنیده اید ولی من از اخبار موثق و مستند گزارشهای اینترنتی و بعضی جاها خوانده ام که بماند ...

- خیلی از این مسئولین فرزندانی در خارج از کشور ندارند
- 
- آنهائی که خارج از کشور فرزندانی دارند و اهمیت نمیدهند مانند محسن رضایی فرزندشان را خارج از کشور کشته اند تا به حکومت شان محکمتر بچسبند.
- 
- آنهائی که فرزندی دارند اصلا مسئول نیستند، یک عده مترسکی هستند که هیچ کاره اند.
- 
- من با سند و مدرک میتوانم ثابت کنم حتی رهبر ایران خامنه ای هم مترسک است و اصلا کاره ای نیست.
- 
بالاتر از رهبری مجمع تشخیص مصلحت نظام و شورای نگهبان و مجلس خبرگان است و همه اینها زیر دست چند حزب هستند. و توی حزبها هم کله گنده ها برای همه تعیین تکلیف میکنند. و این حزبها هستند که نماینده مجلس انتخاب میکنند و تصمیم میگیرند اصلا کسی کشته بشه یا نشه و تحریم یا فیلتر بشه یا نشه، و چون اکثرا بیسواد هستند، اکثرا میان مراکز تصمیم سای نظام بهشون خط میدن، که اکثر این مراکز با نام مراکز تحقیقاتی یا پژوهشکده ها و به خصوص اندیشکده ها شناخته میشوند.  امثال حسن عباسی ها در این مراکز نشسته اند و با چند عدد و رقم و آمار و ارقام و چند حدیث و اظهارنظر کل سرنوشت نظام را تعیین میکنند و رهبری و ارتش و سپاه همه برنامه های اینها را اجراء میکننند.


- تلاشی که میکنید طوری انرژی و وقت و هزینه کنید که نتیجه بدهد نه اینکه اتلاف وقت و انرژی داشته باشید.
- 
- بنابه دلایلی من نمیتوانم بعضی چیزها را در اینجا بگویم چون نیرویهای رژیم اینجا را کاملا رصد میکنند.

اگر واقعا دنبال انجام کار هستید، یک کار تخصصی و درست تر انجام بدهید

پیشنهاد من 

- به جای گرد آوری اطلاعات فرزندان مسئولان مترسک جمهوری اسلامی که هر چند سال یکبار مقام و پست شان عوض میشود و از اداره ای به اداره دیگر میروند، 
- 
اولا آن افراد پشت پرده را به مردم معرفی کنید که استه میان و آسته میرن و کل کشور را آنها میچرخانند،


دوما امار کشته  شده گان و قربانیان خشونت را جمع کنید و برای این افراد وکیل بگیرید و در دادگاههای بین المللی علیه دولت ایران تشکیل پرونده بدهید و وکلایتان اموال جمهوری اسلامی را مصادره کنند و به قربانیان پرداخت کنند. 

البته بارها این کارها انجام شده ولی کسی صدایش را در نیاورده، جمهوری اسلامی نمیگذارد این اخبار منتشر بشه چون تضعیف میشود و رسانه های خارجی هم این اخبار را خیلی بولد نمیکنند چون برایشان منافعی ندارد. و افرادی هم که موفق میشوند خسارتهای سنگین میلیون دلاری بگیرند سکوت میکنند و میروند سر خانه و زندگی شان.

**Reply by samaramco** [reply]
چقدر شکیل و تایپ شده پیام میدید

**Reply by AnonymousMaximus2** [reply]
سالها برنامه نویسی باعث شده اینجوری تایپ کنم تا باگ نداشته باشه، مگه شما برنامه نویس نیستید؟ :)

**Comment by Setareh4IRAN** [comment]
check this link : https://instagram.com/attorneyrazaghi?igshid=YmMyMTA2M2Y=
he is a lawyer working on this topic, he also put an email for people who want to help him :
https://www.instagram.com/p/CjGAlEwr4Th/?igshid=YmMyMTA2M2Y=

Let's make a connection and have the max effect.

**Reply by Setareh4IRAN** [reply]
متوجه نشدم!

**Reply by Setareh4IRAN** [reply]
من ایشون رو پیشنهاد دادم برای پیگیری و همکاری کردن صرفا !
